src/org/ods/orchestration/BuildStage.groovy

```diff
@@ -47,6 +47,10 @@ class BuildStage extends Stage {
                         [Project.TestType.UNIT],
                         data.tests.unit.testResults
                     )
+                    // we check in any case ... (largely because the above call will
+                    // return immediatly when no jira adapter is configured).
+                    // this  will set failedTests if any xunit tests have failed
+                    util.warnBuildIfTestResultsContainFailure(data.tests.unit.testResults)
```

Comment:
@clemensutschig I am not quite sure about this one. Consider that the call to `jira.reportTestResultsForComponent` a few lines above invokes this method already. I think this to be superfluous.

Reply:
Martin.. the warnbuild is idempotent.. and does Not report anythibg to jira.. it parses the Junit Files .. and if failure sets hasfailing tests